WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,15 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
```

Comment:
I assume `showAtAll` is for hiding competitions while you work on setting them up. If that's true, a non visible competition that's over makes no sense. But I'm guessing here.

Anyway, *if* such a competition has somehow been held and there is no delegate report for it, I think the delegate should be notified.